### PR TITLE
Profile connor

### PR DESCRIPTION
**Pull request by connorsicheri**
The profile use case working. This implements the Use case and the interface adapters, along with implementing a data access object to store and retrieve profile data

### CODE REVIEW

File: src/use_case/profile/ProfileDataAccessInterface.java
Comment (lifers) [inline src/use_case/profile/ProfileDataAccessInterface.java:1]: I think interface methods don't need access modifier

File: src/interface_adapter/profile/ProfileState.java
Comment (lifers) [inline src/interface_adapter/profile/ProfileState.java:1]: I didn't see anything that depends on this, is this used in any way? Cleaner to delete this if we don't need it.

File: src/use_case/profile/ProfileOutputBoundary.java
Comment (lifers) [inline src/use_case/profile/ProfileOutputBoundary.java:1]: Interface access modifier; see above.

**Review by lifers — COMMENTED** [review]
The overall code structure looks right, we can merge after we fixed it.

**Review by YoshikuYuu — DISMISSED** [review]
Looks good, but you should change the gamesPlayed variable from double to int

**Review by YoshikuYuu — APPROVED** [review]
Thanks for changing the gamesPlayed variable from a double to an int as requested.
 
Code structure looks clean.
Everything seems correct and it runs properly on my machine, but we should add some JUNIT tests to ensure that everything is correct. Also, we should add some JavaDoc later.
We might need to update the Profile and FileUserDataAccessObject for the Leaderboard use case later on but for now I think we can merge this code into Main.

**Pull request merged**